ocs_ci/helpers/helpers.py

```diff
@@ -3486,10 +3486,15 @@ def get_event_line_datetime(event_line):
          datetime object: The event line datetime
 
     """
-    if re.search(r"\d{4}-\d{2}-\d{2}", event_line):
-        return datetime.datetime.strptime(event_line[:26], "%Y-%m-%d %H:%M:%S.%f")
-    else:
-        return None
+    event_line_dt = None
+    if re.search(r"\d{4}-\d{2}-\d{2}T", event_line):
```

Comment:
please assign `r"\d{4}-\d{2}-\d{2}` to a var and in here (line 3490) append `T`

Reply:
Okay, that will do.